tests/online_ddl/run.sh

```diff
@@ -2,98 +2,93 @@
 
 set -eu
 
-cur=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+cur=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 source $cur/../_utils/test_prepare
 ONLINE_DDL_ENABLE=${ONLINE_DDL_ENABLE:-true}
 BASE_TEST_NAME=$TEST_NAME
 
 function real_run() {
-    online_ddl_scheme=$1
-    run_sql_file $cur/data/db1.prepare.sql $MYSQL_HOST1 $MYSQL_PORT1 $MYSQL_PASSWORD1
-    check_contains 'Query OK, 2 rows affected'
-    run_sql_file $cur/data/db2.prepare.sql $MYSQL_HOST2 $MYSQL_PORT2 $MYSQL_PASSWORD2
-    check_contains 'Query OK, 3 rows affected'
-
-    run_dm_master $WORK_DIR/master $MASTER_PORT $cur/conf/dm-master.toml
-    check_rpc_alive $cur/../bin/check_master_online 127.0.0.1:$MASTER_PORT
-
-    run_dm_worker $WORK_DIR/worker1 $WORKER1_PORT $cur/conf/dm-worker1.toml
-    check_rpc_alive $cur/../bin/check_worker_online 127.0.0.1:$WORKER1_PORT
-    # operate mysql config to worker
-    cp $cur/conf/source1.yaml $WORK_DIR/source1.yaml
-    cp $cur/conf/source2.yaml $WORK_DIR/source2.yaml
-    sed -i "/relay-binlog-name/i\relay-dir: $WORK_DIR/worker1/relay_log" $WORK_DIR/source1.yaml
-    sed -i "/relay-binlog-name/i\relay-dir: $WORK_DIR/worker2/relay_log" $WORK_DIR/source2.yaml
-    dmctl_operate_source create $WORK_DIR/source1.yaml $SOURCE_ID1
-
-    run_dm_worker $WORK_DIR/worker2 $WORKER2_PORT $cur/conf/dm-worker2.toml
-    check_rpc_alive $cur/../bin/check_worker_online 127.0.0.1:$WORKER2_PORT
-    dmctl_operate_source create $WORK_DIR/source2.yaml $SOURCE_ID2
-
-    run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
-        "start-relay -s $SOURCE_ID2 worker2" \
-        "\"result\": true" 1
-
-    # start DM task only
-    cp $cur/conf/dm-task.yaml $WORK_DIR/dm-task-${online_ddl_scheme}.yaml
-    sed -i "s/online-ddl-scheme-placeholder/${online_ddl_scheme}/g" $WORK_DIR/dm-task-${online_ddl_scheme}.yaml
-    dmctl_start_task "$WORK_DIR/dm-task-${online_ddl_scheme}.yaml" "--remove-meta"
-
-    echo "use sync_diff_inspector to check full dump data"
-    check_sync_diff $WORK_DIR $cur/conf/diff_config.toml
-
-    run_sql_file_online_ddl $cur/data/db1.increment.sql $MYSQL_HOST1 $MYSQL_PORT1 $MYSQL_PASSWORD1 online_ddl $online_ddl_scheme
-    run_sql_file_online_ddl $cur/data/db2.increment.sql $MYSQL_HOST2 $MYSQL_PORT2 $MYSQL_PASSWORD2 online_ddl $online_ddl_scheme
-
-    # manually create index to pass check_sync_diff
-    run_sql_tidb "show create table online_ddl.t_target"
-    check_not_contains "KEY \`name\`"
-    run_sql_tidb "alter table online_ddl.t_target add key name (name)"
-
```

Comment:
these lines should not be deleted

Reply:
may something wrong when i rebase master ...